src/USER-SCAFACOS/scafacos.h

```diff
@@ -0,0 +1,74 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef KSPACE_CLASS
+
+KSpaceStyle(scafacos,Scafacos)
+
+#else
+
+#ifndef LMP_SCAFACOS_H
+#define LMP_SCAFACOS_H
+
+#include "kspace.h"
+#include "fcs.h"
```

Comment:
this statement must not be here. as commented before, we have to keep our namespace clean, which means all variables/pointers that require types from a header could be encapsulated into a pointer to a struct (for which we can put a forward declaration into the header) or should be cast to void pointers or stored in compatible regular types and then the header specific types can be used and the header included in the implementation, i.e. the .cpp file only. we had a bunch of issues because of namespace pollution already and it took some effort to get this cleaned up in almost all places where this was possible without having to rewrite large parts of the code (some parts like USER-REAXC, this was only done partially).